source/module_basis/module_ao/ORB_control.cpp

```diff
@@ -32,7 +32,9 @@ ORB_control::ORB_control() : setup_2d(false)
 }
 ORB_control::~ORB_control()
 {
+#ifdef __MPI
```

Comment:
Could you put "#ifdef__MPI and #endif" into the function of Cblacs_exit(1)？

Reply:
Cblacs_exit is an external function and may be difficult to modify